restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/rcml/Parser.java

```diff
@@ -152,6 +162,47 @@ private Tag next() throws LimitExceededException{
         return null;
     }
 
+	///////////////////////////////////////////// new class -- validate
+    private boolean validation(Tag tag) {
+    	boolean ValidationPass=true;
+    	AttributeValidator AttValidator=new Validator();
+    	
+    	switch (tag.name()) {
```

Comment:
i still dont understand why this switch statement is necessary ,and why we need to provide different validator API per RCML verb.I expect a single validator interface, and then different subclasses,invoked with same client code resolved by a map (this is ValidatorFactory responsability).
`
interface AttValidator {
public boolean validate(Tag tag);
}
RecordValidator implements AttValidator{
...
}
ValidatorFactory {
static map<String,AttValidator> validators;
static {
validators.add(new RecordValidator());
...
}
AttValidator retrieveValidator(tag) {
return validators.get(tag.getName());
}
AttValidator validator = validatorFactory.retrieveValidator(tag);
validator,validate(tag);

`